README.md

```diff
@@ -10,16 +10,37 @@ See the site in action at https://www.phoxygen.com
 
 ## Dev workflow
```

Comment:
nit: in English, “title case” means that every word in a title starts with an uppercase letter